src/engine/physics/physics.game.ts

```diff
@@ -115,9 +115,6 @@ export const PhysicsSystem = (state: GameState) => {
 
     if (body) {
       if (!body.isStatic()) {
-        const rotation = Transform.rotation[eid];
-        const quaternion = Transform.quaternion[eid];
-        setQuaternionFromEuler(quaternion, rotation);
```

Comment:
I think we need to get rid of euler rotation on the Transform component entirely. If you want to work in Euler angles you should convert from/to in the spot you want to use it.

Reply:
I am kinda worried that until then this change might cause an issue.

Reply:
this is just for the physics entered query which is only called on creation, it doesn't affect anything else. basically this code was only for setting the rotation of the rigidbody on-creation via the `Transform.rotation` component data, which i don't think we do anywhere right now because all places that instantiate a rigidbody also set position/rotation directly on the rigidbody instead of relying on the Transform component